installer/scripts/genRubyInstaller.sh

```diff
@@ -28,16 +28,16 @@ substitute_arch()
     echo -n "$RESULT"
 }
 
-# This script will generate a "starting point" for an InstallBuilder data file.
-# For example, this generated ruby.data in the omsagent project. This is useful
-# for a package that has an install process (i.e. make install), but we need to
-# generate an install data file for it.
+# This script will generate a file that will contain the new
+# ruby  changes for ruby.data file based on SOURCE_DIR content to be included in ruby install builder 
+# since building ruby has an install process
+# and requires generating an install ruby data file for it.
 
-OUTPUT_RESULTS="/tmp/installBuilder-$USER"
+OUTPUT_RESULTS="/tmp/installBuilder-ruby-data-$USER"
 rm -f $OUTPUT_RESULTS
 
 # Evaluate any wildcards in the filename ...
-SOURCE_DIR=`stat -c "%n" ~/dev/work/oms/intermediate/*/ruby`
+SOURCE_DIR=`stat -c "%n" ../intermediate/*/100/ruby`
```

Comment:
This is making an assumption about your current working directory. If someone doesn't know anything about this script, it will fail for no obvious reason. Before this line, add something like:

```
if [ ! -f configure -o ! -d ../build ]; then
    echo "Must be in OMS build directory for this script to run"
    exit 1
fi

if [ -z "$SOURCE_DIR" ]; then
    echo "Intermediate SSL_100 ruby version does not exist"
    echo "Please build OMS configured with --enable-ulinux"
    exit 1
fi
```

Double check that these 'if' statements do what I expect.